subprojects/tooling-api/src/main/java/org/gradle/tooling/LongRunningOperation.java

```diff
@@ -111,6 +111,16 @@
      */
     LongRunningOperation setJvmArguments(@Nullable String... jvmArguments);
 
+    /**
+     * Appends one or more Java VM arguments to the existing list.
+     *
+     * @param jvmArgument the argument to use for the Gradle process
+     * @param moreJvmArguments more arguments to to use for the Gradle process
+     * @return this
+     * @since 5.0
+     */
+    LongRunningOperation addJvmArguments(@Nullable String jvmArgument, @Nullable String... moreJvmArguments);
```

Comment:
❌ Why the single argument + varargs?

Reply:
❌ Should not be nullable.

Reply:
I'll remove the varargs.

Reply:
It's just easier to specify multiple arguments at once, i.e. `operation.addArguments('--init-script', '/path-to-script')`.

Reply:
Re: nullable: I've only followed the pattern on the existing methods.

Reply:
I don't understand what you mean. `operation.addArguments('--init-script', '/path-to-script') ` doesn't require this two-argument pattern. It should just take varargs. The first parameter doesn't add anything to this method.

The existing methods are setters, which allow you to say "I want you to use the default arguments" by passing `null`. That doesn't make sense for an `add` method.

Reply:
Ok, but if I remove the first parameter clients can call `addArguments()` without arguments which is a no-op.  I just wanted to remove that possibility.

Reply:
Ok, that makes sense. I didn't think about the use case when the client wants to pass through a collection.

Reply:
Done